src/bit-components.js

```diff
@@ -47,6 +47,7 @@ export const AEntity = defineComponent();
 export const Object3DTag = defineComponent();
 export const GLTFModel = defineComponent();
 export const LightTag = defineComponent();
+export const AmbientLight = defineComponent();
```

Comment:
We might want to call this `AmbientLightTag` just to avoid ambiguity and having to rename when importing `AmbientLight` from Three.

Reply:
Updated, thanks